test/sanity/validate-modules/ignore.txt

```diff
@@ -761,6 +761,7 @@ lib/ansible/modules/packaging/os/pacman.py E326
 lib/ansible/modules/packaging/os/pkg5.py E326
 lib/ansible/modules/packaging/os/pkgin.py E322
 lib/ansible/modules/packaging/os/pkgng.py E322
+lib/ansible/modules/packaging/os/pkgutil.py E326
```

Comment:
New ignores should not be added. The underlying issue should be corrected instead. If the issue is in existing module_utils or a docs fragment then the fix should be in a separate PR.

Reply:
The thing is that the documentation only offers **absent**, **latest** and **present** options for the state parameter, but (probably for backward compatibility) the argspec accepts also **installed** and **removed** options.

So we don't want to promote **installed* and **removed**, but I also don't want to break backward compatibility. Hence, ignoring the forced issue.

Reply:
BTW Beats me why the original module didn't have E326 though, it may be a bug in the testing.

Reply:
@dagwieers The **installed** and **removed** state options are new in this PR, which is why the original module passes the tests.

Reply:
Oh, alright, they must have been added by the original author. I'll leave them out.
Sorry for the confusion.